gymnasium/wrappers/rendering.py

```diff
@@ -195,7 +195,7 @@ class RecordVideoV0(
         ...
         >>> env.close()
         >>> len(os.listdir("./save_videos2"))
-        3
+        1
```

Comment:
I agree that this is the correct value it doesn't seem to make sense with the test